Lib/fontmake/font_project.py

```diff
@@ -168,7 +168,7 @@ def build_interpolatable_ttfs(self, ufos, **kwargs):
     def save_otfs(
             self, ufos, ttf=False, interpolatable=False, mti_paths=None,
             is_instance=False, use_afdko=False, subset=True,
-            use_production_names=None):
+            use_production_names=None, **kwargs):
```

Comment:
Instead of allowing any keyword argument here, I think we should exclude `family_name` from args in \__main__.py similar to how we exclude `interpolate`, except between checking glyphs_path and designspace_path.

Maybe make a function which performs the check given an excluded argument name (or better, list of argument names) and allowed source names (to use in the error message), and call that function once with `['family_name'], 'Glyphs'` and then with `['interpolate'], 'Glyphs or MutatorMath'`.